caravel/models.py

```diff
@@ -1574,9 +1577,14 @@ def wrapper(*args, **kwargs):
             d.update(kwargs)
             slice_id = d.get('slice_id', 0)
             slice_id = int(slice_id) if slice_id else 0
+            params = ""
+            try:
+                params = json.dumps(d)
```

Comment:
yes I don't think this is needed anymore, I did this so I could move forward, I think I can turn it off now.